### ISSUE TITLE
Add context menu to enable file logging

### ISSUE THREAD

**Issue by fiveisprime** [Microsoft/vscode-azureappservice#519]
By default, the log files under the Log Files node only have the docker output along with this message

>Logging is not enabled for this container. Please use https://aka.ms/linux-diagnostics to enable logging to see container logs here.

That led me to automatically start right-clicking things to see if I could enable this without starting up streaming and I couldn't find anything. I'm thinking that the context menu should be at the app and Log Files level, but I'm open to discussion on this.

**Comment by fiveisprime** [comment]
Maybe this could be a tree node so it's as obvious as possible? Something like this

![image](https://user-images.githubusercontent.com/1186948/44234852-7e21af00-a15c-11e8-90e0-5e354a8dbaa4.png)

**Comment by nturinski** [comment]
Could we remove the entry points on the File/Log Files node then?  What about the App Service node?

**Comment by fiveisprime** [comment]
Yeah, that's fine. The only ways to connect would be context menu on the root app node or from the nested "connect to" node under "Log Files."
